apps/prairielearn/src/lib/types.ts

```diff
@@ -48,6 +48,7 @@ export type ExpandRecursively<T> = T extends object
     : never
   : T;
 
-export function assertNever(value: never): never {
-  throw new Error(`Unexpected value: ${value}`);
```

Comment:
IMO stack traces make it unnecessary to customize the message here, we can always see where an error originated. A lack of a customizable message is good enough for [Python's `assert_never`](https://docs.python.org/3/library/typing.html#typing.assert_never), which just errors like this:

```
AssertionError: Expected code to be unreachable, but got: 'foo'
```

Reply:
Did that in ed32c0fd05e656102949ea224dc4336fbd3fba0f.